shared/actions/router.js

```diff
@@ -1,42 +1,5 @@
 import * as Constants from '../constants/router'
 
-/*
```

Comment:
@MarcoPolo so i think we need to expose the fact that these nav actions have to do with tabbed routers. i think its kinda dangerous for us to be able to dispatch and have this hidden context (not controlled by code) which is the active tab. I'm leaving the tab parameter optional so nothing breaks but maybe all these actions should go to the tabbed router and it shouldn't be optional?